spec/util_file_spec.rb

```diff
@@ -209,7 +209,7 @@
         it "sets arbitrary header" do
           WebMock.stub_request(:get, uri).to_return(body: "foo", headers: {"Foo" => "Bar"})
           RDF::Util::File.open_file(uri) do |f|
-            expect(f.headers).to include(:foo => %(Bar))
```

Comment:
[`Hurley::Header`](https://github.com/lostisland/hurley/blob/master/lib/hurley/header.rb) does not the enumerable methods (`#all?`, `#any?`) that rspec needs to properly use the `include` matcher. I suspect it's worth following up with them, perhaps, to try to get them in.